ofl/fustat/METADATA.pb

```diff
@@ -0,0 +1,41 @@
+name: "Fustat"
+designer: "Mohamed Gaber"
```

Comment:
Please also add me and Laura Garcia Mut as designers. Cc @Gue3bara @LauraHardtype

Reply:
Apologies, I've added them in https://github.com/google/fonts/pull/7834

Reply:
Oops, wrong GH account.

I've notified Laura about bio and pic and was going to make another PR myself once I have it.
Does it matter when the catalog info gets added? In other words, will you push the data as is for now and we add the profile later?